mlflow/models/evaluation/base.py

```diff
@@ -481,11 +490,19 @@ def evaluate(
         :param run_id: The ID of the MLflow Run to which to log results.
         :param evaluator_config: A dictionary of additional configurations for
                                  the evaluator.
+                                 evaluator_config may contain
+                                 - is_baseline_model: A boolean indicating whether the evaulation
+                                   is for baseline model. For baseline model, no artifact will
+                                   be generated and no metric will be logged.
         :param custom_metrics: A list of callable custom metric functions.
         :param kwargs: For forwards compatibility, a placeholder for additional arguments that
                        may be added to the evaluation interface in the future.
-        :return: An :py:class:`mlflow.models.EvaluationResult` instance containing
-                 evaluation results.
+        :param baseline_model: (Optional) A string URI referring to a MLflow model as baseline model
+                                          to be compared with the candidate model
+                                          for model validation.
+                                          (pyfunc model instance is not allowed)
+        :return: A tuple of :py:class:`mlflow.models.EvaluationResult` instance containing
+                 evaluation results for candidate model and baseline model (if provided else None).
```

Comment:
Could we extend `EvaluationResult` to include information about baseline model performance, rather than returning a tuple of multiple evaluation results where the second element is sometimes None?